src/components/loadingSpinner.jsx

```diff
@@ -0,0 +1,12 @@
+export const LoadingSpinner = () => {
+  return (
+    <div className="fixed top-0 left-0 right-0 bottom-0 w-full h-screen z-50 overflow-hidden bg-gray-700 opacity-75 flex flex-col items-center justify-center">
+      <div
+        role="alert"
+        aria-live="assertive"
+        className="animate-spinner ease-linear rounded-full border-4 border-t-4 border-t-purple border-gray-200 h-12 w-12 mb-4">
+        <p className="invisible">Content is loading...</p>
+      </div>
+    </div>
+  );
+};
```

Comment:
My idea here was to create an overlay with some opacity that goes over the form and makes the spinner visible ... I failed on the overlay part.

Reply:
I see :thinking: this is just a thought, and I don't know what others think, but what if the spinner was over the form component alone? without the overlay.

Or, could we move the spinner below the search bar where the result has to appear? That way the user knows where the result will be when it arrives?...

Thoughts @adropofilm and @ctoffanin ?

Reply:
It might be interesting. I'm gonna try it tomorrow.

Reply:
Ok just looked over the deployment, the spinner is nice! 

I think it makes sense to put it below the search, just so the user has an idea of where to look once the results are rendered.

Reply:
I tried to fix it a little.
Check if it does make sense.
@Njong392 @adropofilm

Reply:
> I tried to fix it a little. Check if it does make sense. @Njong392 @adropofilm

ok ok. Reviewed. Soooo sorry if it seems like I am over-reviewing this, but what if the spinner was smaller, and aligned left? That is, it was under the search bar, but to the left at the edge. How would that look? @adropofilm

Reply:
No please, there is no over-reviewing. That actual one is 12px.
I'm going to drop another one 8px and on the left. 
We can compare them and eventually find a good one.

Reply:
I just added 2 more spinners, 1 size 8px and 1 size 10px.
The smaller stands on the left, and the middle one is just there in between to compare the sizes and eventually pick one.

Reply:
The 8px one looks good to me!

Reply:
Updated

Reply:
I thought that was what Fatima meant.
To avoid fetching, if the value of the search is the same as the last user's input.
Did I misunderstand?

Reply:
`clearDataBeforeFetch()` is a function I made on top. I needed to use the same content as the `useEffect` and to avoid repetition I create a function to reuse.

Reply:
> I thought that was what Fatima meant. To avoid fetching, if the value of the search is the same as the last user's input. Did I misunderstand?

Fatima was saying that after the first fetch, if the user submits the same slang, the result should be provided without another request being sent.

Reply:
@Njong392  Oh wait, now maybe I get what you mean with:

> Oops, there seems to be a problem. If I search say idk the first time, and the loader comes, after that I get results. But when I search the same thing the second time, the response doesn't come back at all. 

Do you mean you re-type the same thing and not submit it again without typing? right?
If that is the case, I'll take a look.